### PR TITLE
typescript

### PR DESCRIPTION
**Pull request by shreyas-jadhav**
also uses wavesurfer 5

### CODE REVIEW

**Comment by ShiiRochi** [comment]
@shreyas-jadhav, thanks for your help. I'll check asap.

**Comment by ShiiRochi** [comment]
@shreyas-jadhav, do you have some WIP or can I consider, that it's ready?

**Comment by shreyas-jadhav** [comment]
Hey, I don't have any WIP for now. You can check it out.

File: package.json [hunk 1]
```diff
@@ -1,39 +1,38 @@
 {
   "name": "wavesurfer-react",
-  "version": "2.0.12",
+  "version": "2.0.13",
```
Comment (ShiiRochi) [inline package.json:3]: Version should not be changed, it is controlled with npm cli

**Review by ShiiRochi — CHANGES_REQUESTED** [review]
1. tgz file should be removed and added to the ignored list
2. version of the package shouldn't be changed manually

**Review by ShiiRochi — CHANGES_REQUESTED** [review]
Drop commits that removes and adds tarball file and do the same stuff with commits related to version in `package.json`. I don't want to store that information within repository and git history. You can do that stuff with creating dev branch from current master and cherry picking all commits until those with tarball and version updates. If they are commited all together (i.e. with changes) then do soft reset to current (in my repo) master state make hard force push.

**Comment by ShiiRochi** [comment]
@shreyas-jadhav, I closed this PR because I've already done what I wrote here some time ago. I cherry-picked only required stuff from your repo branches and you can see that Git saved all credits properly, i.e. the contribution of yours is seen, otherwise I would have waited until your response.

**Pull request closed without merging**